conans/test/integration/toolchains/meson/test_mesontoolchain.py

```diff
@@ -517,3 +518,19 @@ def generate(self):
     client.save({"conanfile.py": conanfile})
     client.run("install .", assert_error=True)
     assert "You can only pass native=True if you're cross-building" in client.out
+
+
+def test_compiler_path_with_spaces():
+    conanfile = (GenConanfile().with_name("lib").with_version("1.0").with_generator("MesonToolchain")
+                 .with_settings("compiler"))
```

Comment:
```suggestion
    conanfile = GenConanfile().with_name("lib").with_version("1.0").with_generator("MesonToolchain")
                .with_settings("compiler")
```

Reply:
This doesn't work in Python, I didn't notice it at first. I fixed with another way to avoid parenthesis.

Reply:
yes, you have to add a `\`, it's fine to keep the `()`